### ISSUE TITLE
Label of tool for comment in python does not always work

### ISSUE THREAD

**Issue by simonaubertbd** [amphi-ai/amphi-etl#178]
Hello,

Sometimes, it doesn't work

![378400873-4b1ab189-f15a-4800-b3e9-d8b242b2f3e8](https://github.com/user-attachments/assets/1a7023da-8811-4b1c-ba91-01a9d1ad29ca)

I think it only works when you have overwrite the default name. Also, maybe the tool id would help?

Best regards,

Simon

**Comment by tgourdel** [comment]
Yeah that was on purpose, I assumed if no custom title were provided it meant the user didn't want to add more information and hence just keep the boilerplate comments. I think it would be a bit redondant in most cases?
Like in your example of Summary you'd have:
```
# Summary
# Generate summary statistics
```

Let me know what you think

**Comment by simonaubertbd** [comment]
@tgourdel  Ah ok not a bug but a feature ^^

What I had in mind would be
`# generated by tool id {id of the too} : Summary
# Generate summary statistics`

The generated by is important, because it acts like a tag for research... and since several tools can have the same label, the id would be cool too.

Best regards,

Simon

**Comment by simonaubertbd** [comment]
Hello @tgourdel I close this. It's solved by #390
